### PR TITLE
Social Media, My Avatar and Mentor Avatar

### PR DESCRIPTION
**Pull request by NaviRocker**
Social Media Feeds Added, Added my Avatar and Added a Mentor Avatar
Solutions for Three Tasks in one

### CODE REVIEW

**Comment by rishiraj824** [comment]
@naveentherocker Please give a testing link for reviewers to review your PR. Also, squash your commits into one. Thanks!

**Comment by NaviRocker** [comment]
https://naveentherocker.github.io/gci17.fossasia.org/

**Comment by NaviRocker** [comment]
Please see only Social Media 2. 

Social Media is not the correct one. It is over written by Social Media 2

**Comment by abishekvashok** [comment]
@naveentherocker can you decrease the blank space in mobile view by making the youtube section responsive as well.

screenshot from 2017-11-30 17-23-28

**Comment by SaptakS** [comment]
@naveentherocker hey. Great work. Just a suggestion. Why not use one of the already present sections in the website for the social media rather than creating it entirely new. Say maybe the section which already has a video, use that for the youtube embedding. Then maybe the section below that use for the other social media. Just a suggestion.

**Comment by NaviRocker** [comment]
Ok sure

**Comment by NaviRocker** [comment]
@abishekvashok done

**Comment by NaviRocker** [comment]
@SaptakS  It will be available in the Final Website for sure

**Comment by NaviRocker** [comment]
Now I have made all three Changes as mentioned in the PR Topic

**Comment by NaviRocker** [comment]
https://naveentherocker.github.io/gci17.fossasia.org/

File: index.html [hunk 1]
```diff
@@ -284,17 +327,49 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 		        
 		    </section><section class="bg-secondary">
 		        <div class="container">
-		            <div class="row mb64 mb-xs-24">
-		                <div class="col-sm-12 text-center spread-children-large">
```
Comment (SaptakS) [inline index.html]: I see you have deleted the images and code added by other students. That is not acceptable. If you are making a change in style, you can't just add your information and delete other's

File: index.html [hunk 2]
```diff
@@ -284,17 +327,49 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
...
+		                   
+
+		                <div class="col-md-3 col-sm-6">
+		                    <div class="image-tile outer-title text-center">
+		                        <img alt="Pic" src="img/naveen.jpg">
+		                        <div class="title mb16">
+		                            <h5 class="uppercase mb0">Naveen Rajan</h5>
+		                            <span>GCI Participant</span>
+		                        </div>
+		                        <p class="mb0">
```
Comment (SaptakS) [inline index.html]: Code indentation is completely out of place

File: index.html (hunk 1 above)
Comment (NaviRocker) [reply index.html]: Oh very sorry i didn't knew it. When i already submitted and it says that there is a Conflict.

File: index.html [hunk 3]
```diff
@@ -284,17 +327,49 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
...
+ <ul class="list-inline social-list mb24">
```
Comment (SaptakS) [inline index.html]: Please indent your code properly.

File: index.html [hunk 4]
```diff
@@ -13,7 +13,24 @@
         <link href="css/theme-red.css" rel="stylesheet" type="text/css" media="all" />
         <link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
         <link href='http://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600' rel='stylesheet' type='text/css'>
-    </head>
+    <style>
+	.video-container {
+    position: relative;
```
Comment (SaptakS) [inline index.html]: Indentation out of place.

Comment (SaptakS) [reply index.html]: Also you should put all the custom css styles that you are implementing in the css/custom.css file.

**Review by SaptakS — CHANGES_REQUESTED** [review]
Please check my inline comments. Also request you to remove the .db files from your commit.

**Comment by SaptakS** [comment]
Also there is a merge conflict and there are 12 commits. Please squash your commits. If you don't have idea about squashing, you can read [here](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)

**Review by NaviRocker — COMMENTED** [review]
Done

File: index.html [hunk 5]
```diff
@@ -1,10 +1,9 @@
----
----
```
Comment (abishekvashok) [inline index.html]: These are required for jekyll

File: index.html [hunk 6]
```diff
@@ -1,10 +1,9 @@
...
+
 <!doctype html>
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title>FOSSASIA | Google Code-In 2017</title>
```
Comment (abishekvashok) [inline index.html]: I think the old title was finer than no title 😂

File: index.html [hunk 7]
```diff
@@ -27,7 +27,8 @@
 		        <div class="nav-bar">
 		            <div class="module left">
 		                <a href="index.html">
-		                    <img class="logo logo-dark" alt="FOSSASIA" src="img/fossasia-dark.png">
```
Comment (abishekvashok) [inline index.html:30]: Fossasia logo

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Always try to fetch from upstream.

File: index.html [hunk 8]
```diff
@@ -1,10 +1,10 @@
----
----
+----
```
Comment (abishekvashok) [inline index.html]: Three --- are enough.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by NaviRocker** [comment]
Any Changes?

**Comment by NaviRocker** [comment]
@SaptakS  @abishekvashok  Every Suggestions changed

**Comment by SaptakS** [comment]
@naveentherocker please squash your commits. I have already provided a link above that will help you in squashing.

**Comment by abishekvashok** [comment]
Mariobehling is already added @naveentherocker. Also squats your commits:

`git rebase -i HEAD~noofcommitstosquash`

A text editor will open, change the all "pick"s to 'squash' except the first one, and save and exit. Then the editor will open again for you to specify the commit message, specify and save and exit. Then force push,

`git push -f`

**Comment by NaviRocker** [comment]
I am using Github Desktop Application. How to do it from that?

**Comment by abhinavk96** [comment]
@naveentherocker it will be a great time for you to  get familiar with git. try.github.io has excellent tutorials for beginners. Spending an hour or two learning will be really worth your while and really help you out in the future as a developer :)

**Comment by NaviRocker** [comment]
Thank You @CosmicCoder96  and Can i use Git Shell for this?

**Comment by abhinavk96** [comment]
@naveentherocker yup!

**Review by abishekvashok — COMMENTED** [review]
You still got an image of mariobehling. And thumbs.db

File: index.html [hunk 9]
```diff
@@ -3,8 +3,8 @@
 <!doctype html>
 <html lang="en">
     <head>
-        <meta charset="utf-8">
-        <title>FOSSASIA | Google Code-In 2017</title>
+		<meta charset="utf-8">
```
Comment (abishekvashok) [inline index.html]: Let's properly intend the code.

File: index.html [hunk 10]
```diff
@@ -183,6 +185,11 @@ <h2 class="mb-xs-16">Meet Foundry, the cleverest way<br class="hidden-sm"> to bu
 		        </ul>
 		    </section><section class="bg-secondary">
 		        <div class="container">
+
+		            
+
```
Comment (abishekvashok) [inline index.html]: unwanted blank lines

File: index.html [hunk 11]
```diff
@@ -307,17 +314,26 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 		        
 		    </section><section class="bg-secondary">
 		        <div class="container">
+
+
```
Comment (abishekvashok) [inline index.html]: blank lines

File: index.html [hunk 12]
```diff
@@ -307,17 +314,26 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
...
+		           
+
 		            <div class="row mb64 mb-xs-24">
 		                <div class="col-sm-12 text-center spread-children-large">
 		                    <img class="col-sm-2" alt="Miguel Piedrafita" src="img/m1guelpf.png">
 		                    <img class="col-sm-2" alt="Chahat Bansal" src="img/chahat.png">
 		                    <img class="col-sm-2" alt="Nitesh Kumar Niranjan" src="img/niteshkumarniranjan.jpg">
 		                    <img class="col-sm-2" alt="Aleksandr Lymarenko" src="img/a-v-lymarenko.png">
-		                    <img class="col-sm-2" alt="Pic" src="img/avatar4.png">
+		                    <img class="col-sm-2" alt="Pic" src="img/naveen.jpg">
```
Comment (abishekvashok) [inline index.html]: Use your name as a value for the alt attribute.

File: index.html [hunk 13]
```diff
@@ -307,17 +314,26 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
...
 		                    <img class="col-sm-2" alt="Pic" src="img/avatar5.png">
+
 		                </div>
-		            </div>
-		            
+		                <div class="col-md-4 col-sm-6">
+
```
Comment (abishekvashok) [inline index.html]: blank lines, empty div

File: index.html [hunk 14]
```diff
@@ -888,4 +930,4 @@ <h6 class="title">Instagram</h6>
         <script src="js/scripts.js"></script>
     </body>
 </html>
-				
+
```
Comment (abishekvashok) [inline index.html]: Add a newline at EOF without any whitespaces.

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Some more changes, after which you should squash your commits and this PR would be ready to go.

**Comment by NaviRocker** [comment]
Its not getting Squashed. Why is it?

**Comment by abishekvashok** [comment]
@naveentherocker did you force push after squashing?
What commands did you run?
Give me the output of `git rebase -i HEAD~23`
(23 since there are 23 commits now in this pr)

**Comment by NaviRocker** [comment]
Corrected Everything

**Comment by NaviRocker** [comment]
image

This is what i get @abishekvashok

**Comment by abishekvashok** [comment]
@naveentherocker pls see my commands, I can still see blank lines and intedation faults, let's lkeep this professional. :) Can you correct them pls? Then squash the commits.

**Comment by abishekvashok** [comment]
change every pick except the first one to "squash" (without quotes), then save and exit, then on the another popup, delete everything and enter your commit message in this format:
```
Commit title

Commit message
```

**Comment by NaviRocker** [comment]
There are Extra Commits in this file rather than my ones

**Comment by abishekvashok** [comment]
As i can see there are 24 commits of your own in this PR. Ignore the rest of the commits, rewrite the history by moving them down the list. (don't delete them though).

**Comment by NaviRocker** [comment]
image
Is this Fine?

**Comment by abishekvashok** [comment]
@naveentherocker try it. Save and exit. Then force push. Better make a backup of your repo first.

**Comment by NaviRocker** [comment]
This must be used there know?

**Comment by NaviRocker** [comment]
git push -f origin gh-pages

**Comment by abishekvashok** [comment]
@naveentherocker I meant to save and exit the textpad, then on the command line, `git push -f origin gh-pages`git push -f origin gh-pages

**Comment by NaviRocker** [comment]
Yeah I did like that

**Comment by NaviRocker** [comment]
image

**Comment by abishekvashok** [comment]
@naveentherocker pls abort the rebase by running `git rebase --abort`. Then try to chery pick the commits:

reference: https://stackoverflow.com/questions/35123108/cherry-pick-and-squash-a-range-of-commits-into-a-subdirectory-or-subtree
https://stackoverflow.com/questions/7212976/how-to-squash-my-changes-into-a-single-result-commit-and-pick-into-master-branch
http://alblue.bandlem.com/2011/06/git-tip-of-week-cherry-picking.html

**Comment by NaviRocker** [comment]
Done

**Comment by abishekvashok** [comment]
@naveentherocker still I can see 25 commits.

**Comment by NaviRocker** [comment]
Problem Still Continues. Please Squash and Merge

**Comment by SaptakS** [comment]
@naveentherocker @abishekvashok The squashing has been taking quite a lot of time and there have been internal merge conflicts in your commits which is causing conflicts on rebase.

@naveentherocker do these:

```
git reset --soft HEAD~25
git commit -m "<Your commit message>"
git push -f origin gh-pages
```

This should solve the problem.

**Comment by abishekvashok** [comment]
Conflicts.

**Review by SaptakS — APPROVED** [review]

**Comment by SaptakS** [comment]
@naveentherocker you have conflicts in the 4 above mentioned files. So what you need to do is 
* open this 4 files
* search for `>>>>>>>`
* you will see something like
```
<<<<< gh-pages
/* code1 */
=======
/* code 2 */
>>>>>> gh-pages
```
* you need to decide on what part of code1 and code to keep and delete the arrow and equal sign lines.

Please let me know if I am clear enough

**Comment by NaviRocker** [comment]
Solved

**Review by isuruAb — APPROVED** [review]
👍

**Comment by NaviRocker** [comment]
@abishekvashok Waiting for your review

**Review by abishekvashok — APPROVED** [review]

**Comment by abishekvashok** [comment]
@naveentherocker LGTM, Good work!

**Pull request merged**